### DISCUSSION TITLE
void on plane override antipad

### DISCUSSION THREAD

**Discussion by giuseppeselli** [ansys/pyaedt#3857] [Q&A] [answered]
greetings all,

I am building a via structure using pyaedt, I have a void associated with a trace that connects a via that has an antipad. Regardless of wether I instantiate the via before or after the void, the void seems to always override the antipad of the via. So what happens is that the antipad shrinks to the size of the void of the trace connecting the via. Is there a way to avoid this ?

Thanks,
GS

**Comment by Samuelopez-ansys** [comment]
@svandenb-dev @ring630 Any idea?

**Comment by hui-zhou-a** [comment, marked as answer]
@giuseppeselli you can set below property to True.

`edb.design_options.antipads_always_on = True`

**Reply by giuseppeselli** [reply]
excellent ! it works !

**Reply by giuseppeselli** [reply]
thank you very mcuh
